source/containers.tex

```diff
@@ -5840,7 +5846,9 @@
 \end{codeblock}
 
 \rSec2[associative.set.syn]{Header \tcode{<set>} synopsis}%
-\indexlibrary{\idxhdr{set}}
+
+\indextext{\idxhdr{set}}%
```

Comment:
Escaping the newline is a bit pointless if the whole thing is surrounded by whitespace :-S

I would mildly suggest to not bother escaping newlines for index entries in between blocks... eh, I don't really care, it's just a bit of a pointless change.